test/common/testlib.py

```diff
@@ -122,14 +133,18 @@ def __init__(self, address, label, machine, pixels_label=None, port=None):
         self.default_user = "admin"
         self.label = label
         self.pixels_label = pixels_label
+        self.current_layout = "mobile" if bool(os.getenv("TEST_MOBILE", "")) else "desktop"
```

Comment:
I'd store the scenario dict here, rather than the name.  It'll make things easier everywhere else, particularly if you store the `is_mobile` flag in the dict (see below).

Reply:
Yes, good idea!